src/rules/sumArray.js

```diff
@@ -0,0 +1,44 @@
+
+import { get, isArray, isNil, isObject, reduce } from 'lodash';
+import { isFloat } from 'validator';
+
+function evaluateMin(value, min) {
+  if (isNil(min) || min === '') {
+    return false;
+  }
+  return !isFloat(value.toString(), { min: parseFloat(min) });
+}
+
+function evaluateMax(value, max) {
+  if (isNil(max) || max === '') {
+    return false;
+  }
+  return !isFloat(value.toString(), { max: parseFloat(max) });
+}
+
+export default function sumArray(field, value, options) {
+  if (!isArray(value)) {
+    return null;
```

Comment:
If value is not an array, shouldn't be returned string `sumArray` as on the line 30? From my point of view, when a variable `number` is not float, that means, we have some corrupted value... the same could be valid for this line :)